### DISCUSSION TITLE
Does Vercel have a feature similar to AWS Lambda layers?

### DISCUSSION THREAD

**Discussion by behrangsa** [vercel/community#1004] [Help] [answered]
Some Node packages don't contain their binary dependencies (e.g. https://www.npmjs.com/package/node-latex).

Is there a way to provide these dependencies in a zip file &mdash; for example &mdash; that gets decompressed into the `$PATH`?

**Comment by amyegan** [comment, marked as answer]
Vercel does not currently offer a feature like Lambda Layers for Serverless Functions. But the team is always working to add new capabilities. If you want to work push more for this feature, [the research team is asking for feedback](https://github.com/vercel/community/discussions/1002) that will be used to guide future plans 🙇

**Comment by rondo10** [comment]
> Some Node packages don't contain their binary dependencies (e.g. https://www.npmjs.com/package/node-latex).
> 
> Is there a way to provide these dependencies in a zip file &mdash; for example &mdash; that gets decompressed into the `$PATH`?

I'm very curious how we can overcome this obstacle.
I'm using ImageMagick and need the binary to be installed on the production environment.
There's something new on this subject?

**Reply by gavindoughtie** [reply]
This is the current issue I'm working through (for texlive-pdf and its associated `pdflatex` command)
